pipelines/tasks/get-package-versions-list.yaml

```
@@ -0,0 +1,51 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: get-package-versions-list
+spec:
+  volumes:
+    - name: unpinned-packages
+      emptyDir:
+        medium: Memory
+    - name: script
+      configMap:
+        name: get-packages-script
+        defaultMode: 0755
+  workspaces:
+    - name: pinned-packages
+  params:
+    - name: PACKAGES
+      type: array
+  stepTemplate:
+    volumeMounts:
+      - name: unpinned-packages
+        mountPath: /scratchspace
+      - name: script
+        mountPath: /scripts
+  steps:
+    - image: registry.access.redhat.com/ubi9-micro
+      name: format-unpinned-packages-to-file
+      command: /scripts/get-packages
```

`command` is an array of strings:

```suggestion
      command: [/scripts/get-packages]
```